ipykernel/kernelbase.py

```diff
@@ -217,6 +217,9 @@ def _parent_header(self):
         "shutdown_request",
         "is_complete_request",
         "interrupt_request",
+        # I have the impression that there is amix/match debug_request and do_debug_request
+        # former was from ipyparallel but is marked as deprecated, but now call do_debug_request
+        # "do_debug_request"
```

Comment:
Maybe not leave this in the codebase?

Reply:
Well, it's an actual question I have, and I don't think it does belong in an issue. 
I'm  not too sure where to leave it, and I'm certain if I remove it, I will spend  again an hour to figure that out next time I try to clean this up.

Reply:
`debug_request` is not related to ipyparallel. ipyparallel doesn't use or support the debugging messages.

Reply:
Then something in wrong on like 228 of the file (before modifications), 
it does state that some of those are ipyparallel deprecated mesages:

https://github.com/ipython/ipykernel/blob/2ca799276f4458d50085bb1ef8e2ddeb08a283a6/ipykernel/kernelbase.py#L223-L233

Reply:
Yes, the comment is outdated, it only applies to the first two message types here. From `debug_request` onward are control-channel-only messages. #1282 removes the deprecated messages and fixes the comment.